_notices/rsn0044.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 44 # should match notice number
+notice_pin: true # set to true to pin to notice page
+
+title: "Moving older RAPIDS conda packages to a `legacy` label"
+notice_author: RAPIDS Ops
+notice_status: Proposal
+notice_status_color: blue

```

Comment:
Needs to be yellow since we plan to do this but haven't completed it yet. We don't really use "Proposal."
```suggestion
notice_status: In Progress
notice_status_color: yellow
```